common/repo/tests/TestTool.java

```diff
@@ -35,6 +35,7 @@
     this.repoFactory = repoFactory;
   }
 
+  @SuppressWarnings({"checkstyle:LeftCurly", "checkstyle:Indentation"})
```

Comment:
@oferb `checkstyle` got confused because of the brackets in `case` statement (which were put there to introduce a new scope). I'd change the whole thing to `if-else-if` chain (which also introduces scope but looks much cleaner). Let's address this in next PR.

Reply:
Ok